metricbeat/module/kibana/cluster_actions/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"kibana",
+        "name":"actions",
```

Comment:
should this be `"name":"cluster_actions"` ?

Reply:
Yup, bad copy/paste here